config/src/network_id.rs

```diff
@@ -311,4 +337,41 @@ mod test {
         );
         assert_eq!(expected, serde_yaml::to_string(&context).unwrap());
     }
+
+    #[derive(Debug, Deserialize, Eq, PartialEq, Serialize)]
+    #[serde(rename = "NetworkId", rename_all = "snake_case")]
+    enum OldNetworkId {
+        Validator,
+        #[allow(dead_code)]
```

Comment:
do you still need this annotation?

Reply:
No, I'll remove it.  I originally wasn't testing public